libraries/BluetoothSerial/src/BluetoothSerial.cpp

```diff
@@ -241,6 +243,7 @@ static void esp_spp_cb(esp_spp_cb_event_t event, esp_spp_cb_param_t *param)
         }
         xEventGroupClearBits(_spp_event_group, SPP_DISCONNECTED);
         xEventGroupSetBits(_spp_event_group, SPP_CONNECTED);
+        xEventGroupSetBits(_spp_event_group, SPP_CONGESTED);
```

Comment:
Why don't you roll this up into one `xEventGroupSetBits()`
```c++
 xEventGroupSetBits(_spp_event_group, SPP_CONNECTED | SPP_CONGESTED );
```

Reply:
Yes, you are right. To be honest, I did not give it much thought. I do not think it makes a great difference in performance nor in code readability, but I can change it if necessary for merging.